### PR TITLE
P0/notes crud

### PR DESCRIPTION
**Pull request by purvasharmadev**
## Added Notes feature

- user can create posts
- user can edit a post
- delete a post
- add tags, priority to notes

### CODE REVIEW

**Review by adityapandadev — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,17 @@
 import "./App.css";
 import URLRoutes from "./routes";
+import {Nav} from "./Pages/Nav/nav";
+import {Footer} from "./Pages/Footer/footer";
+
```
Comment (itsmebipul00) [inline src/App.js:5]: Sugggestion: Could be exported from an index file

File: src/Component/EditNote/edit-note.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  });
+
+  const getNote = () => {
+    setNote(state.notesList.find((item) => item._id === id));
+
+  };
+
+  useEffect(() => {
+    getNote();
+    console.log(note)
```
Comment (itsmebipul00) [inline src/Component/EditNote/edit-note.js:23]: remove console log

**Review by itsmebipul00 — APPROVED** [review]
Good to merge

File: src/App.css [hunk 3]
```diff
@@ -1,10 +1,46 @@
 :root{
   --PRIMARY-COLOR:#ffde59;
-  --SECONDARY-COLOR:red;
-  --TEXT-COLOR:#2d1adb;
+  --SECONDARY-COLOR:rgb(245, 233, 233);
```
Comment (sumitk-singh) [inline src/App.css:3]: Suggestion: keep the consistency, either use **hex** code or **rgb**

File: src/routes.js [hunk 4]
```diff
@@ -11,6 +11,7 @@ import { RestrictedRoute } from "./Auth/AuthRoutes/RestrictedRoutes";
 
 import { HomePage } from "./Pages/Home/home-page";
 import { Notes } from "./Pages/Notes/notes";
```
Comment (sumitk-singh) [inline src/routes.js:13]: you may use index.js file in the same directory for import export.
Like:  import {HomePage, Notes} from './Pages/index.js';

**Review by sumitk-singh — APPROVED** [review]
Good usage of class name,
Folder structure is organized.
clean and readable code. Good to merge with some small changes as mentioned.

**Pull request merged**